cyberark-secrets-provider-for-kubernetes/values.schema.json

```diff
@@ -0,0 +1,126 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "required": [
+    "rbac",
+    "secretsProvider",
+    "environment"
+  ],
+  "properties": {
+    "rbac": {
+      "required": [
+        "roleName",
```

Comment:
@abrahamko @eladkug we should check min and max length of these field for k8s. Is 1 character enough to be valid?

Reply:
Will a space (ascii, etc) character be a valid service/role name?

Reply:
I disagree.
Let k8s do its own validations, I think we shouldn't do validations on behalf of k8s.

Reply:
I think it is done automatically without need to code